code/__DEFINES/clothing.dm

```diff
@@ -98,7 +102,8 @@
 #define HAND_LEFT	512
 #define HAND_RIGHT	1024
 #define HANDS		1536
-#define FULL_BODY	2047
+#define NECK		2048
+#define FULL_BODY	4096 //2047

```

Comment:
This needs to be 4095 to fulfill the functionality of the old flag. Here's the reason:
* Old FULL_BODY corresponded to bitflag (111 1111 1111 - meaning it's considered to cover every bodypart);
* New FULL_BODY corresponds to bitflag (1 0000 0000 0000), which mean it **no longer is considered to cover the entire body (or anything, really), logic-wise**;